data/yam/agama/auto/sle_ext4.jsonnet

```diff
@@ -0,0 +1,57 @@
+{
+  product: {
+    id: '{{AGAMA_PRODUCT_ID}}',
+    registrationCode: '{{SCC_REGCODE}}'
+  },
+  user: {
+    fullName: 'Bernhard M. Wiedemann',
+    password: '$6$vYbbuJ9WMriFxGHY$gQ7shLw9ZBsRcPgo6/8KmfDvQ/lCqxW8/WnMoLCoWGdHO6Touush1nhegYfdBbXRpsQuy/FTZZeg7gQL50IbA/',
+    hashedPassword: true,
+    userName: 'bernhard'
+  },
+  root: {
+    password: '$6$vYbbuJ9WMriFxGHY$gQ7shLw9ZBsRcPgo6/8KmfDvQ/lCqxW8/WnMoLCoWGdHO6Touush1nhegYfdBbXRpsQuy/FTZZeg7gQL50IbA/',
+    hashedPassword: true
+  },
+  storage: {
+    drives: [
+      {
+        alias: 'pvs-disk',
+       }
+    ],
+    volumeGroups: [
+      {
+        name: 'system',
+        physicalVolumes: [
+          { generate: ['pvs-disk'] }
+        ],
+        logicalVolumes: [
+          {
+            size:   { 'default' },
+            filesystem: { path: '/', type: 'ext4' }
+          },
+          {
+            size:   { 'default' },
```

Comment:
it should be a way to only specify this partition change because the other are default from the product, perhaps removing them. We also should not put any size to the partitioning and leave it with the default.